rust-wasm-id-allocator/distributed-id-allocator/src/compressor.rs

```diff
@@ -73,10 +86,17 @@ impl IdCompressor {
         }
     }
 
+    /// Returns the current sizing used for new clusters.
     pub fn get_cluster_capacity(&self) -> u64 {
         self.cluster_capacity
     }
 
+    /// Updates the sizing used for new cluster creation.
```

Comment:
I'd add a warning about only setting this after consensus has been reached on the value or else divergence will occur.